[Bug]: Error in 3.5.1

### What happened?

Addon is not supporteded in blender 3.5.1

Blender 3.5.1 had som important bug fixes, so if this addon coulde be updated it woulde be great! 

### Add-on Version

2.5.0

### Blender versions

_No response_

### Relevant log output

```Shell
[ERROR] exception: out of range
[ERROR] injector detected the unsupported version of Blender. Please upgrade Drag-and-Drop Support or downgrade Blender
[ERROR] detected version : 3.5.1
```

--- a/src/blender-injection/BlenderPatchPatterns.h
+++ b/src/blender-injection/BlenderPatchPatterns.h
@@ -205,5 +205,13 @@ static std::unordered_map<std::string, BlenderPatchPattern> Patchers{
             "E8 7B FF FF FF 84 C0",
             "E8 0B BC 00 00 48 85 C0"
         }
+    },
+    {
+        "3.5.1", {
+            "E8 AE 4D 3F 00 84 DB",
+            "4C 8D 05 09 08 00 00",
+            "E8 7B FF FF FF 84 C0",
+            "E8 0B BC 00 00 48 85 C0"
+        }
     }
 };

